src/legacy/config/config-set.spec.ts

```diff
@@ -790,7 +790,6 @@ describe('_resolveTsConfig', () => {
       expect(conf.options.configFilePath).toBeUndefined()
       expect(findConfig).not.toHaveBeenCalled()
       expect(readConfig.mock.calls[0][0]).toBe('/foo/tsconfig.bar.json')
-      expect(parseConfig).not.toHaveBeenCalled()
```

Comment:
this will now be called to find out if we there are project references in the tsconfig to check.